XRTK.SDK/Packages/com.xrtk.sdk/Editor/SDKPackageInstaller.cs

```diff
@@ -36,15 +37,30 @@ private static void ImportPackageAssets()
 
         private static void CheckPackage()
         {
+            var updateProfileGUIDs = false;
+            var updatePrefabsGUIDs = false;
 
             if (!EditorPreferences.Get($"{nameof(SDKPackageInstaller)}.Profiles", false))
             {
-                EditorPreferences.Set($"{nameof(SDKPackageInstaller)}.Profiles", PackageInstaller.TryInstallAssets(HiddenProfilePath, $"{DefaultPath}\\Profiles"));
+                updateProfileGUIDs = PackageInstaller.TryInstallAssets(HiddenProfilePath, $"{DefaultPath}\\Profiles", false);
+                EditorPreferences.Set($"{nameof(SDKPackageInstaller)}.Profiles", updateProfileGUIDs);
             }
 
-            if (!EditorPreferences.Get($"{nameof(SDKPackageInstaller)}.Prefabs", false))
+            if (updateProfileGUIDs && !EditorPreferences.Get($"{nameof(SDKPackageInstaller)}.Prefabs", false))
             {
-                EditorPreferences.Set($"{nameof(SDKPackageInstaller)}.Prefabs", PackageInstaller.TryInstallAssets(HiddenPrefabPath, $"{DefaultPath}\\Prefabs"));
+                updatePrefabsGUIDs = PackageInstaller.TryInstallAssets(HiddenPrefabPath, $"{DefaultPath}\\Prefabs", false);
+                EditorPreferences.Set($"{nameof(SDKPackageInstaller)}.Prefabs", updatePrefabsGUIDs);
+            }
+
+            if (updateProfileGUIDs && updatePrefabsGUIDs)
+            {
+                PackageInstaller.RegenerateGuids(new[] { $"{DefaultPath}\\Profiles", $"{DefaultPath}\\Prefabs" });
+            }
+            else
+            {
+                //Unset prefs as one of the copies has failed.
+                EditorPreferences.Set($"{nameof(SDKPackageInstaller)}.Profiles", false);
+                EditorPreferences.Set($"{nameof(SDKPackageInstaller)}.Prefabs", false);
```

Comment:
This re-wrote all my guids just by opening the project without a key set. People may run into this same issue when they clone the repo the first time.

Reply:
https://github.com/XRTK/SDK/pull/174#discussion_r420257430